eoc_journal/eoc_journal.py

```diff
@@ -196,6 +208,65 @@ def _get_current_anonymous_user_id(self):
         """
         return self.runtime.anonymous_student_id
 
+    def get_progress_metrics(self):
+        """
+        Fetches and returns dict with progress metrics for the current user
+        in the course.
+        """
+        user = self._get_current_user()
+        course_id = self._get_course_id()
+        client = ApiClient(user, course_id)
+
+        metrics = client.get_user_progress_metrics()
+        
+        if metrics:
+            user_progress = metrics.get('completions', 0)
+            cohort_average = metrics.get('course_avg', 0)
+
+            return {
+                'user': round(user_progress),
+                'cohort_average': round(cohort_average),
+            }
+        return None
+
+    def get_engagement_metrics(self):
+        """
+        Fetches and returns dict with engagement metrics for the current user
+        and course.
+        """
+        user = self._get_current_user()
+        course_id = self._get_course_id()
+        client = ApiClient(user, course_id)
+
+        user_engagement = client.get_user_engagement_metrics()
+        course_engagement = client.get_cohort_engagement_metrics()
+
+        if not course_engagement:
+            return None
+        else:
+            course_point_sum = [
+                calculate_engagement_score(metrics)
+                for metrics in course_engagement['users'].itervalues()
+            ]
+            course_point_sum = sum(course_point_sum)
+
+            enrollments = course_engagement['total_enrollments']
+
+            if enrollments > 0:
+                cohort_score = float(course_point_sum) / enrollments
+            else:
+                cohort_score = 0
+
+            return {
+                'user_score': calculate_engagement_score(user_engagement),
+                'cohort_score': cohort_score,
```

Comment:
Please round `user_score` and `cohort_score`. I am seeing: "The cohort average is 16.6666666667"